app/views/admin/loans/_logs.html.slim

```diff
@@ -11,7 +9,7 @@ javascript:
   $(function() {
     new MS.Views.LogListView({
       el: 'section.log-list',
-      refreshUrl: #{json refresh_url},
+      refreshUrl: #{json "/admin/logs?project=#{@loan.id}"},
```

Comment:
Not sure why the `org` stuff is able to be deleted here or why it was here in the first place, but I'm trusting you, and this is cleaner!